RecoTauTag/HLTProducers/test/README.md

```diff
@@ -0,0 +1,8 @@
+Instruction to run the L2TauNNTag test
+cd CMSSW_12_1_0_pre3/src
+cmsenv
+scram b -j 10
+cmsRun testL2TauTagNN.py
```

Comment:
- add steps on how to extract di-tau path from confdb for completeness
- add formatting